### PR TITLE
CDM-24: Create Bullet Point component

### PR DESCRIPTION
**Pull request by danielaprado**
### Description

- Create BulletPoint component
- Desktop & Mobile responsive

### Attachments
- **Desktop with one element**
desktop

- **Desktop with more than one element**
desktop_2

- **Mobile with one element**
mobile

- **Mobile with more than one element**
mobile_2

### Reviewers

- [ ] @catarinapm 
- [x] @mariasbl

### CODE REVIEW

File: fe/app/components/BulletPoint/BulletPoint.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { BulletPointProps } from "@/app/components/BulletPoint/BulletPoint.models";
+
+const BulletPoint = ({ category, text }: BulletPointProps) => {
+  return (
+    <div className="flex sm:gap-3 gap-2 items-baseline pt-5">
+      <div
+        className={`flex-shrink-0 w-2 h-2 sm:w-[10px] sm:h-[10px] bg-${category}`}
```
Comment (mariasbl) [inline fe/app/components/BulletPoint/BulletPoint.tsx:7]: Can you please update the breakpoint to md? 😊

File: fe/app/components/BulletPoint/BulletPoint.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      />
+      <p className="text-xs sm:text-base">{text}</p>
```
Comment (mariasbl) [inline fe/app/components/BulletPoint/BulletPoint.tsx:9]: Can you please add lato font family to the text? 😊

File: fe/app/components/BulletPoint/BulletPoint.tsx (hunk 1 above)
Comment (danielaprado) [reply fe/app/components/BulletPoint/BulletPoint.tsx:7]: Sure!

File: fe/app/components/BulletPoint/BulletPoint.tsx (hunk 2 above)
Comment (danielaprado) [reply fe/app/components/BulletPoint/BulletPoint.tsx]: Yes!

**Review by mariasbl — APPROVED** [review]

**Pull request merged**